content/download/upgradeRecipe/upgradeRecipe8.adoc

```diff
@@ -1638,3 +1638,13 @@ After in `*ConstraintProvider.java*`:
 
   ...
 ----
+
+== From 8.9.0.Final to 8.10.0.Final
```

Comment:
In the upgrade recipe, should we tell them to upgrade from Quarkus 1.x to Quarkus 2.x?